specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,861 @@
+            },
+            "environmentId": {
+              "description": "Resource ID of environment.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "configuration": {
+              "$ref": "#/definitions/JobConfiguration",
+              "description": "Container Apps Job configuration properties."
+            },
+            "template": {
+              "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+              "description": "Container Apps job definition."
+            },
+            "outboundIpAddresses": {
+              "description": "Outbound IP Addresses of a container apps job.",
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "readOnly": true
+            },
+            "eventStreamEndpoint": {
+              "description": "The endpoint of the eventstream of the container apps job.",
+              "type": "string",
+              "readOnly": true
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      },
+      "required": []
+    },
+    "ContainerAppsJobPatchProperties": {
+      "type": "object",
+      "description": "ContainerApp resource specific properties.",
+      "properties": {
+        "identity": {
+          "description": "Managed identities needed by a container app job to interact with other Azure services to not maintain any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "Provisioning state of the Container Apps Job.",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled",
+                "Deleting"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "ContainerAppsJobProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "environmentId": {
+              "description": "Resource ID of environment.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "configuration": {
+              "$ref": "#/definitions/JobConfiguration",
+              "description": "Container Apps Job configuration properties."
+            },
+            "template": {
+              "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+              "description": "Container Apps job definition."
+            },
+            "outboundIpAddresses": {
```

Comment:
>outboundIpAddresses [](http://example.com/codeflow?start=13&length=19)

PATCHable properties can't be readOnly.

Reply:
Turns out it's actually allowed so payload definition can be shared across PUT and PATCH, but in this case where it's just PATCH, it doesn't make sense.